osu.Game/Online/Multiplayer/MultiplayerClient.cs

```diff
@@ -36,6 +36,21 @@ public abstract partial class MultiplayerClient : Component, IMultiplayerClient,
         /// </summary>
         public virtual event Action? RoomUpdated;
 
+        /// <summary>
+        /// Invoked when a user's local style is changed.
+        /// </summary>
+        public event Action<MultiplayerRoomUser>? UserStyleChanged;
+
+        /// <summary>
+        /// Invoked when a user's local mods are changed.
+        /// </summary>
+        public event Action<MultiplayerRoomUser>? UserModsChanged;
```

Comment:
https://github.com/ppy/osu/pull/32250#discussion_r1998365964

> Is this granularity required? As far as I can tell, both of these methods have two subscribers each, and each of the subscriptions in each pair does exactly the same thing.
> 
> `UserGameplaySettingsChanged` maybe?

If it's not too big of a deal, can we roll with this under the basis that it matches every other event here (corresponds 1-1 with a server event), except `RoomUpdated`?

Reply:
Sure.